src/Xml/XmlDocument.php

```diff
@@ -48,79 +44,121 @@
 use const LIBXML_ERR_WARNING;
 
 /**
- * Class for validating OPUS import xml.
+ * A class for loading and validating XML
+ * (replaces/merges two previous separate validation classes).
```

Comment:
Der Kommentar in der Klammer ist gut gemeint, aber hilft niemanden in der Zukunft. Macht eher Sinn als Teil einer Commit-Message.

Reply:
Habe ich entfernt.